docs/markdown/Using-wraptool.md

```diff
@@ -82,3 +82,13 @@ straightforward:
 Wraptool can do other things besides these. Documentation for these
 can be found in the command line help, which can be accessed by
 `meson wrap --help`.
+
+## Automatic dependency fallback
+
+Since *0.64.0* Meson can use WrapDB to automatically find missing dependencies.
+
+The user simply needs to download latest database:
+    $ meson wrap update-db
+
+Once the database is available locally, any dependency not found on the system
+but available in WrapDB will automatically be downloaded.
```

Comment:
The path `subprojects/wradb.json` file should be mentioned here, too. Having it only in release notes has poor discoverability.

Reply:
done